vmdb/app/models/miq_request_workflow.rb

```diff
@@ -957,8 +957,10 @@ def get_ems_metadata_tree(src)
       @ems_xml_nodes = {}
       xml = MiqXml.newDoc(:xmlhash)
       convert_to_xml(xml, result)
-      _log.info "Load EMS metadata for: <#{@ems_xml_nodes.keys.inspect}>"
-      _log.info "EMS metadata collection completed in [#{Time.now - st}] seconds"
+      if _log.debug?
+        _log.info "#{log_header} Load EMS metadata for: <#{@ems_xml_nodes.keys.inspect}>"
+      end
+      _log.info "#{log_header} EMS metadata collection completed in [#{Time.zone.now - st}] seconds"
```

Comment:
There is no need for `log_header` when using _log.  That is causing test failures since log_header is no longer defined here.